AntWars/Board/Ants/Ant.cs

```diff
@@ -54,7 +54,7 @@ internal Ant(Board board, Player owner, int viewRange, int maxInventory, int mov
             this.UnitsGone = 0;
             Owner = owner;
             Inventory = 0;
-            MoveRange = moveRange;
+            MoveRange = moveRange * board.diagonal;
```

Comment:
das ist doof, wenn wir einen constructor haben wo moverange drin steht und im object ein attribute haben das Moverange heißt gehe ich davon aus das das 1zu1 so übernommen wird. das hier halte ich für verwirrend. und sollte eigentlich schon so in die ameise gegeben werden